.env.example

```diff
@@ -3,16 +3,18 @@
 # THIS IS A SAMPLE ENCRYPTION KEY AND SHOULD NEVER BE USED FOR PRODUCTION
 ENCRYPTION_KEY=6c1fe4e407b8911c104518103505b218
 
+# Required
+DB_CONNECTION_URI=postgres://infisical:infisical@db:5432/infisical
```

Comment:
Should `@db` be `@localhost` so we can run the schema migrations?

Reply:
We can't do that because this .env.example is being used for docker compose example. 

The docker compose doesn't expose db port via localhost